rust/private/repository_utils.bzl

```
@@ -324,6 +331,7 @@ def BUILD_for_rust_toolchain(
         llvm_profdata_label = llvm_profdata_label,
         extra_rustc_flags = extra_rustc_flags,
         extra_exec_rustc_flags = extra_exec_rustc_flags,
+        opt_level_arg = opt_level_arg,
```

I think this cleans up the formatting a bit
```suggestion
        opt_level = repr(opt_level),
```